### DISCUSSION TITLE
blog/content-writing

### DISCUSSION THREAD

**Discussion by giscus** [GawdOfGithub/dcc-blog#3] [General]
# blog/content-writing

https://blog.dccnita.in/blog/content-writing

**Comment by GawdOfGithub** [comment]
This is working!!!

**Comment by slash-aech** [comment]
Is it okay even if the story which we compose in task 1 (Storytelling) is fictional? Or is it supposed to be strictly real?

**Comment by slash-aech** [comment]
And also with the hook (task 3) is it supposed to be text based only? What does "more clickable" in the evaluation criteria refers to?
